### DISCUSSION TITLE
[Feature request] Artisan command to generate Trait

### DISCUSSION THREAD

**Discussion by vsahinid** [laravel/framework#43054] [Ideas]
I would like to work on an artisan command to generate a Laravel Trait class using `php artisan make:trait TraitName` which would create a predefined file in the directory `app/Http/Traits`.  Just wanted to see if this would be ok before tackling it.  Thanks!

**Comment by henzeb** [comment]
This is something an IDE should do. I can only see the use case of such a command for things that are laravel specific.
